torch/onnx/utils.py

```diff
@@ -1314,6 +1314,70 @@ def _validate_dynamic_axes(dynamic_axes, model, input_names, output_names):
             dynamic_axes[key] = value_dict
 
 
+# This function export torch::jit::Graph object
+# to serialized ONNX ModelProto.
+# In addition to the Python tests of this function
+# (found by searching
+# _export_jit_graph_to_onnx_model_proto in Pytorch
+# codebase), this function can be called in C++ to
+# to generate ModelProto without accessing disk.
+# For a concrete example, see how we create and call
```

Comment:
This seems like it should be in a test in the lazy tensor code base, not in a comment here. I'd leave this out and you can provide this example to whoever needs it when they need it.

Reply:
removing from this PR